yt_dlp/YoutubeDL.py

```diff
@@ -1459,7 +1459,10 @@ def extract_info(self, url, download=True, ie_key=None, extra_info=None,
 
             temp_id = ie.get_temp_id(url)
             if temp_id is not None and self.in_download_archive({'id': temp_id, 'ie_key': key}):
-                self.to_screen(f'[{key}] {temp_id}: has already been recorded in the archive')
+                self.to_screen(f'[{key}] '
+                               f'{self._format_screen(temp_id, self.Styles.ID)}: '
+                               'has already been recorded in the archive'
+                               )
```

Comment:
Maybe inline the second f-string? Also closing brackets should reside on the same line.
```suggestion
                self.to_screen(f'[{key}] {self._format_screen(temp_id, self.Styles.ID)}: '
                               'has already been recorded in the archive')
```
Maybe also prepend `entry` so its the same as the other message format? (`entry has already been recorded in the archive`)